13_sandboxes/test_case_generator.py

```diff
@@ -0,0 +1,274 @@
+# ---
+# cmd: ["modal", "run", "-m", "13_sandboxes.test_case_generator::main"]
+# ---
+import modal
+
+app = modal.App(name="sandbox-test-case-generator")
+model_volume = modal.Volume.from_name("deepseek-model-volume", create_if_missing=True)
+files_volume = modal.Volume.from_name("files-volume", create_if_missing=True)
+
+
+model_image = (
+    modal.Image.from_registry("nvidia/cuda:12.4.0-devel-ubuntu22.04", add_python="3.12")
+    .pip_install(
+        "transformers==4.53.2",
+        "torch==2.7.1",
+        "accelerate==1.8.1",
+        "hf_transfer==0.1.9",
+    )
+    .env(
+        {
+            "HF_HUB_ENABLE_HF_TRANSFER": "1",
+            "HF_HOME": "/cache",
+        }
+    )
+    .entrypoint([])  # silence noisy logs
+)
+
+with model_image.imports() as imports:
+    from transformers import AutoModelForCausalLM, AutoTokenizer
+
+
+# GH Repo Configs
+GH_OWNER = "modal-labs"
+GH_REPO_NAME = "password-analyzer"
+GH_MODULE_NAME = "password_strength"
+GH_BRANCH = "main"
+
+
+@app.cls(
+    image=model_image,
+    volumes={
+        "/cache": model_volume,
+        "/data": files_volume,
+    },
+    gpu="L40S",
+)
+class Deepseek:
```

Comment:
Naming nit that IMO is actually pretty deep -- this isn't really a `DeepSeek` class. It's more like a `DeepSeekPythonTestWriter` or a `PythonTestWriter`.

Maybe that's how you can think of what goes in the system prompt: the system prompt directs the model to generally behave as indicated by the class name. User prompts describe the specific behavior/provide the relevant context for it to serve their needs.